src/Stacktrace.php

```diff
@@ -137,7 +137,12 @@ protected function breakUpTheStacks()
             $newMessage = $newMessage[0];
             preg_match_all(static::REGEX_STACK_MESSAGE, $newMessage, $match);
 
-            $this->message = $match[1][0];
+            foreach ($match as $subMatch) {
```

Comment:
I'm not opposed at all to this change, I just worry about things going not-displayed. Do you have any ideas on a way to prevent things from going through the cracks? ☹️

Reply:
So I looked into it some more and for this case, we can actually solve this issue while also maintaining some kind of message. (not defaulting to an empty string).

```php
$this->message = $match[1][0] ?? $newMessage
```

Since the `$match[1][0]` bit doesn't seem to always be there, the `$newMessage` bit should always be a string, and thus should have some good useful information.

I'd like to go with this newer solution using the null coalesce operator for simplicity, if you don't mind 😅.

Reply:
Fixed in c92215d